eve/main.yml

```diff
@@ -2,7 +2,7 @@
 version: "0.2"
 
 branches:
-  documentation/*, feature/*, improvement/*, bugfix/*, w/*, q/*, hotfix/*:
+  documentation/*, feature/*, improvement/*, bugfix/*, w/*, q/*, hotfix/*, user/*:
```

Comment:
should this really be commited ?

Reply:
of course it should, we forgot to have the `user/*` branches covered in this repo. I needed to add it each time I needed to run the cI. :laughing: